AutomatedTesting/Gem/PythonTests/atom_renderer/test_Atom_GPUTests.py

```diff
@@ -40,6 +43,34 @@ def golden_images_directory():
     return golden_images_dir
 
 
+def create_zip_archive(archive_path):
+    """
+    Creates a new zip file archive at archive_path containing all files listed within archive_path.
+    :param archive_path: location containing the files to archive, the zip archive file will also be saved here.
+    :return: None, but creates a new zip file archive inside path containing all of the files inside archive_path.
+    """
+    files_to_archive = []
+
+    # Search for .png and .ppm files to add to the zip archive file.
+    for (folder_name, sub_folders, file_names) in os.walk(archive_path):
+        for file_name in file_names:
+            screenshot_regex = "(.*png$|.*ppm$)"
+            if bool(re.search(screenshot_regex, file_name)):
```

Comment:
nit: you could just do `file_name.endswith(".png") and file_name.endswith(".ppm")`. This will mostly be faster than the regex